docs/root/configuration/cluster_manager/cluster_stats.rst

```diff
@@ -10,8 +10,9 @@ General
 -------
 
 The cluster manager has a statistics tree rooted at *cluster_manager.* with the following
-statistics. Any ``:`` character in the stats name is replaced with ``_``. Some of the stats
-concerning number of clusters may include statically specified xDS servers in the count.
+statistics. Any ``:`` character in the stats name is replaced with ``_``. Stats concerning
+number of clusters will count xDS servers if the Envoy (rather than Google) gRPC client
+implementation is used.
```

Comment:
How about rephrasing to "Stats include all clusters managed by the cluster manager, including both clusters used for data plane upstreams and control plane xDS clusters.". I still think it's a bit redundant, but it doesn't hurt to state.